content/blog/2023-04-18-the-value-of-composability-in-ethereum-classic/index.md

```diff
@@ -0,0 +1,90 @@
+
+![A single database.](./etc-composability.png)
+
+From a developer perspective, composability is also extremely valuable because all the ecosystem of smart contracts inside ETC becomes a collection of services that may be reused and leveraged, thus reducing development time and increasing innovation.
+
+All the components and services inside ETC are like lego blocks that both users and developers may put together as they please. This has enabled a very unique experience in this type of technology.
+
+This level of composability is very likely an unintended benefit not anticipated by Satoshi Nakamoto when he invented Bitcoin, although he did envision that it would have smart contracts eventually, an idea that proved very difficult come true due to that system’s design.
+
+## Composability vs Interoperability
+
+The heart of composability in ETC and other smart contract networks is the replicated database, which is what we know as the blockchain. 
+
+The blockchain is a ledger that stores accounts, balances, and smart contracts, and it is replicated because each time a new transaction or smart contract is sent to the network it is distributed and copied in every node of the system. 
+
+This feature was originally intended to provide a high degree of redundancy for security reasons, but also proved to have additional benefits such as composability.
+
+Interoperability, on the other hand, is the ability of different blockchains to interact with one another. This is distinct from composability because it needs everything inside the same network, while interoperability is the ability of doing things with resources and services from different isolated networks.
+
+Therefore, composability is lost when services are hosted in different blockchains. 
+
+## Composability vs Layering
+
+Similar to interoperability, layering also affects composability. 
+
+Layering is similar to interoperability, but instead of looking to interact with other blockchains, it seeks to increase scalability of transactions.
+
+If a blockchain is limited to more or less 1.5 million transactions per day, then it falls short of consumer level needs that run in the billions per day. Layering basically moves the high volume and low value transactions out of the base layer blockchain to higher layer systems that process these quantities more efficiently and then settle final netted transactions on the lower layer unscalable system.
+
+As may be guessed, just for the fact that transactions, thus some dapps or their components, will be moved to higher layer systems to satisfy transaction volumes, composability will be affected as everything is not inside the same network.
+
+Ethereum Classic has this kind of constraint, it will be able to comfortably handle up to about 1.5 million transactions per day, but if users need all of their things to be done at the base layer, then they will likely end up paying higher fees in exchange for the security guarantees.
```

Comment:
Line 57: Fix run-on sentence
```suggestion
Ethereum Classic has this kind of constraint. It will be able to comfortably handle up to about 1.5 million transactions per day, but if users need all of their things to be done at the base layer they will likely end up paying higher fees in exchange for the security guarantees.
```